<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelersMainWindow</class>
 <widget class="QMainWindow" name="TravelersMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TravelersMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
color: blue; 
 padding-top: 5px;
 padding-right: 5px;
 padding-bottom: 5px;
 padding-left: 5px;
border: 2px solid #8f8f91;
border-radius: 6px;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
min-width: 80px;

}


 </string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>361</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">text: center;
</string>
         </property>
         <property name="text">
          <string>                                European Cities </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_cities"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Distance From Berlin: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="distanceLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Traditional Food Items from: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cityLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_items"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_back">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_NextCity">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next City </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="makeCustomTripPage">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="makeCustomTripLayout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>160</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="makeCustomTripButtonLayout">
       <item>
        <widget class="QStackedWidget" name="makeCustomTripButtonPages">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="blankPage"/>
         <widget class="QWidget" name="buttonPage">
          <widget class="QPushButton" name="makeCustomTripButtonConfirm">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>2</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Select Initial City</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page404">
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>129</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="text">
          <string>Please Select Only One City!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushErrorButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="makeCustomSubTripLayout">
       <item>
        <widget class="QListWidget" name="checkSubsequentCities"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>160</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="makeCustomSubTripButtonLayout">
       <item>
        <widget class="QStackedWidget" name="makeCustomSubTripButtonPages">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="blankPage_3"/>
         <widget class="QWidget" name="buttonPage_3">
          <widget class="QPushButton" name="makeCustomSubTripButtonConfirm">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>2</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Select Additional Cities</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>139</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="errorLabel_2">
         <property name="text">
          <string>Select At Least One Additional City!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="subsequentTripErrorButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="completedCustomTrip"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Trip Starting London</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Visit the Initial 11 Cities</string>
    </property>
   </widget>
   <widget class="QPushButton" name="makeCustomTripButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Custom Trip</string>
    </property>
   </widget>
   <widget class="QListWidget" name="checkCitiesWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>409</width>
      <height>239</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdmin_Login"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdmin_Login">
   <property name="text">
    <string>Admin Login</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
